Align wc LC with core (#3643)
SAP/luigi: docs/luigi-compound-container-api.md

-   **since**: 1.0.0

### documentTitle

The document title value to be passed to the compound micro frontend.

Type: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)

**Meta**

-   **since**: NEXT_RELEASE

### hasBack

The hasBack value to be passed to the compound micro frontend.

> You can use it to mark if there is one or more preserved views. Useful when needing to show a back button.

Reviewer: ```suggestion
It indicates that there is one or more preserved views. Useful when you need to show a back button.
```

Type: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)

**Meta**

-   **since**: NEXT_RELEASE

### dirtyStatus

The dirty status value to be passed to the compound micro frontend.

> Used for unsaved changes when navigating away

Reviewer: ```suggestion
It's used to indicate that there are unsaved changes when navigating away.
```
